src/odemis/acq/stream/_base.py

```diff
@@ -1279,12 +1280,40 @@ def getPixelCoordinates(self, p_pos: Tuple[float, float]) -> Optional[Tuple[int,
         pixel_pos_c = tform.inverse().apply(p_pos)
         # MD_POS is the center of the image, so subtract half of the size to convert to pixel-coordinates
         # A "-" is used for the y coordinate because Y axis has the opposite direction in physical coordinates
-        pixel_pos = (int(math.floor(pixel_pos_c[0] + size[0] / 2)),
-                     int(math.floor(- (pixel_pos_c[1] - size[1] / 2))))
-        if 0 <= pixel_pos[0] < size[0] and 0 <= pixel_pos[1] < size[1]:
-            return pixel_pos
-        else:
-            return None
+        pixel_pos = (pixel_pos_c[0] + size[0] / 2,
```

Comment:
Why not do the `int` and `math.floor` here itself like before and do it in line 1286?